packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/App.xaml.cs

```diff
@@ -21,6 +21,12 @@ public App()
             MainComponentName = "SampleApp";
             JavaScriptMainModuleName = "index.windows";
 
+#if DEBUG
+            InstanceSettings.UseWebDebugger = true;
+#else
+            InstanceSettings.UseWebDebugger = false;
+#endif
```

Comment:
I would expect that it must be not in code, but rather passed as a command line parameter.

Reply:
I've just found that there is a parameter, but it's not working. #3601